Config.uk

```diff
@@ -0,0 +1,5 @@
+config LIBPYTHON_SHAPELY
+    bool "Shapely - Python package for planar geometric objects"
+    default n
+    select LIBPYTHON_NUMPY
+    select LIBGEOS
```

Comment:
I would add `select X` for all the necessary dependencies, it would be difficult to keep track of them otherwise.

Reply:
I believe the one direct dependency that's missing is python3, as shapely directly makes use of python API headers; I'll add it to the list.
As for other, indirect dependencies, I strongly believe it's better to delegate to the depended-on libraries instead of aggregating duplicate information here.